Makefile

```diff
@@ -121,9 +135,18 @@ ifneq ($(HPC_LIBS_DIR),)
 		--build-arg "HPC_LIBS_DIR=$(HPC_LIBS_DIR)" \
 		-t $(DOCKERHUB_REGISTRY)/$(NGC_PYTORCH_HPC_REPO)-ss:$(SHORT_GIT_HASH) \
 		.
-ifneq ($(HPC_TMP_LIBS_DIR),)
-	rm -rf $(HPC_LIBS_DIR)
-endif
+        ifneq ($(HPC_TMP_LIBS_DIR),)
+	    rm -rf $(HPC_LIBS_DIR)
+        endif
+        ifeq "$(BUILD_SIF)" "1"
+	    @echo "BUILD_SIF: $(NGC_PYTORCH_HPC_REPO)-ss:$(SHORT_GIT_HASH)"
+	    make build-sif TARGET_NAME="$(NGC_PYTORCH_HPC_REPO)-ss:$(SHORT_GIT_HASH)"
+        endif
+else
+        ifeq "$(BUILD_SIF)" "1"
+	    @echo "BUILD_SIF: $(NGC_PYTORCH_HPC_REPO):$(SHORT_GIT_HASH)"
+	    make build-sif TARGET_NAME="$(NGC_PYTORCH_HPC_REPO):$(SHORT_GIT_HASH)"
```

Comment:
I'm not sure having ":" in the sif name would be a good idea. Something that was mentioned in the slack was
ngc-pt-24.10-hpc- and adding x86/arm64. Currently it creates sif name like "pytorch-ngc-hpc-dev:d47dc18.sif"